watertap/costing/watertap_costing_package.py

```diff
@@ -252,7 +252,7 @@ def build_process_costs(self):
         )
         self.total_operating_cost = pyo.Var(
             initialize=1e3,
-            domain=pyo.NonNegativeReals,
+            domain=pyo.Reals,
             doc="Total operating cost",
```

Comment:
This is where I changed the domain in the costing package and it creates some different solutions in the testing files below, one of which has a difference larger than 10%. @bknueven @adam-a-a